pkg/shed/db.go

```diff
@@ -101,35 +101,36 @@ func (db *DB) Put(key, value []byte) (err error) {
 // Get wraps LevelDB Get method to increment metrics counter.
 func (db *DB) Get(key []byte) (value []byte, err error) {
 	value, err = db.ldb.Get(key, nil)
-	if errors.Is(err, leveldb.ErrNotFound) {
-		db.metrics.GetNotFoundCounter.Inc()
-		return nil, err
-	} else {
+	db.metrics.GetCounter.Inc()
+	if err != nil {
 		db.metrics.GetFailCounter.Inc()
+		if errors.Is(err, leveldb.ErrNotFound) {
+			db.metrics.GetNotFoundCounter.Inc()
+		}
+		return nil, err
 	}
-	db.metrics.GetCounter.Inc()
 	return value, nil
 }
 
 // Has wraps LevelDB Has method to increment metrics counter.
 func (db *DB) Has(key []byte) (yes bool, err error) {
 	yes, err = db.ldb.Has(key, nil)
+	db.metrics.HasCounter.Inc()
```

Comment:
you might be incrementing twice here, which is not the purpose of this counter originally. also the changes below do the same. not sure why this is needed